server/image_list.go

```diff
@@ -33,16 +33,20 @@ func ConvertImage(from *storage.ImageResult) *types.Image {
 		return nil
 	}
 
-	repoTags := []string{"<none>:<none>"}
+	repoTags := []string{}
 	if len(from.RepoTags) > 0 {
 		repoTags = from.RepoTags
 	} else if from.PreviousName != "" {
 		repoTags = []string{from.PreviousName + ":<none>"}
 	}
 
-	repoDigests := []string{"<none>@<none>"}
-	if len(from.RepoDigests) > 0 {
+	repoDigests := []string{}
+	if len(from.RepoDigests) > 0 { // nolint: gocritic
 		repoDigests = from.RepoDigests
+	} else if from.Name != "" {
```

Comment:
AFAICS `from.Name` is a name:tag or a name@digest, so this is invalid to concatenate with a @ digest.